### ISSUE TITLE
Jet appointments: Allow for provider (not user) to set automatic booking of all future slots once a slot is booked

### ISSUE THREAD

**Issue by rinibak** [CrocoBlock/suggestions#7926]
The reason for this is that I'm a therapist and I always see patients on the same and time every week, this is standard. So if someone books a Monday at 10, then I want that Monday at 10 appointment to no longer be available for the following weeks, so that no one goes and books it, as then I will be double-booked, which will create chaos in my practice.

The way the plugin is set right now, a user can book a slot and that's it, or they can book a recurring appointment, but they choose this themselves.

I would like you to create a provider setting to work in such a way as to allow the following: if someone books, say, a monday at 10, then all Mondays at 10 from that week on will also be booked. **This feature should be allowed for me, the provider, not the user. I don't want to give the user this option, but I do want the appointment to be a recurring appt.**

So, in summary, I need to be able as the provider to have a setting that allows for automatically booking all future weeks of a slot when a user books that slot. They book a slot and automatically, all same day/time slots are booked for the following weeks/months, until I as the provider go and make the slot available again (say, when the patient ends the therapy).

I have already asked for this in a 20 minute call with one of your support staff (some 2 months ago) and I have also asked for it in chat. Please make this a priority, as it is extremely necessary in my case. 

Many thanks

**Comment by Crocoblock** [comment]
Unfortunately, we do not plan to introduce such functionality into our plugin. Now our team is working on more priority tasks. However, in the future, we will not exclude the possibility of adding this functionality to the core of our plugin.

Thank you for being so understanding.
